.travis.yml

```diff
@@ -24,9 +26,11 @@ matrix:
     - env: MODE=breaking
     - go: master
 
+before_install:
+  - curl -L -o $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v$DEP_VERSION/dep-linux-amd64 && chmod +x $GOPATH/bin/dep
```

Comment:
we should add here a check on the SHA of the binary to make sure it is what we expect . for 0.4.1
it is  `31144e465e52ffbc0035248a10ddea61a09bf28b00784fd3fdd9882c8cbb2315` 

somethign like  :
```
- curl -fL -o $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v$DEP_VERSION/dep-linux-amd64  && echo "$DEP_SHA $GOPATH/bin/dep" | sha256sum -c - && chmod +x $GOPATH/bin/dep
```

Reply:
https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64.sha256

Reply:
This will verify the hash?